com.qzandronum.Q-Zandronum.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Kevin Degeling -->
+
+<component type="desktop">
+  <id>com.qzandronum.Q-Zandronum</id>
+  <name>Q-Zandronum</name>
+  <summary>A Zandronum 3.0 fork with improved netcode, configurable movement and many small tweaks</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+
+  <developer_name>Q-Zandronum &amp; Doomseeker team</developer_name>
+  <update_contact>kreato@kreato.dev</update_contact>
+
+  <url type="homepage">https://qzandronum.com/</url>
+  <url type="bugtracker">https://github.com/IgeNiaI/Q-Zandronum/issues</url>
+  <url type="help">https://github.com/IgeNiaI/Q-Zandronum/wiki</url>
+
+  <launchable type="desktop-id">com.qzandronum.Q-Zandronum.desktop</launchable>
+
+  <description>
+    <p>
+	    Q-Zandronum is a fork of Zandronum, a multiplayer oriented port, based off Skulltag, for Doom and Doom II by id Software.
+	    The main goal of the fork is to improve the QC:DE gameplay, but it can be used with other mods as well. The engine brings improved netcode, multiple Quality-of-Life improvements and new modding features.
+    </p>
+    <p>
+      In some scenarios, commercial data files are required to run the supported games. In other cases, it's possibly to automatically use Freedoom as a drop-in replacement. This package comes bundled with Doomseeker for easy installation and server browsing.
+    </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/com.zandronum.Zandronum/master/images/image_01.png</image>
```

Comment:
this doesn't exist. the build will fail.

Reply:
How should i do this then?
Can i just specify local files like
`<image>images/image_01.png</image>`?

I couldn't find any documentation about how should i approach this.

Reply:
simply the URL is 404 Not Found. So you need to figure out where they should be. Usually upstream is a good place.

Reply:
the URL does load for me, but i will change it anyway.

Reply:
https://github.com/flathub/flathub/pull/4475/commits/d5655733da6875239774d4d11f82c9b43b27106a will this work?

Reply:
oh it's loading them from a **different** package (I didn't notice at first). This is doomed to fail at one point since it's not pinned to a specific revision.